jvector-base/src/main/java/io/github/jbellis/jvector/util/PoolingSupport.java

```diff
@@ -1,11 +1,10 @@
 package io.github.jbellis.jvector.util;
 
+import java.util.concurrent.LinkedBlockingQueue;
```

Comment:
no performance change vs MpmcAQ?

Reply:
We only use it to track new values, its not in hot path anymore (see #127 )